java/sage/MediaFile.java

```diff
@@ -5486,6 +5489,29 @@ public String execVideoThumbGen(String pathString, boolean deinterlace, float of
       int targetStream)
   {
     List<String> args = new ArrayList<String>(25);
+    String res;
+    synchronized (this)
```

Comment:
Synchronizing around 'this' doesn't really have much purpose here as it's dealing with static variables. You can just drop the synchronization altogether since it's completely safe to run the test multiple times in parallel because they'll end up setting them to the same value.